sourcetracker/_cli/gibbs.py

```diff
@@ -142,11 +143,12 @@ def gibbs(table_fp, mapping_fp, output_dir, loo, jobs, alpha1, alpha2, beta,
     # failed if so.
     os.mkdir(output_dir)
 
-    # Load the metadata file and feature table. Do a data check on the
-    # feature_table.
+    # Load the metadata file and feature table.
     sample_metadata = parse_sample_metadata(open(mapping_fp, 'U'))
-    _ft = load_table(table_fp)
-    feature_table = biom_to_df(_ft, apply_fractional_value_correction=True)
+    feature_table = biom_to_df(load_table(table_fp))
```

Comment:
A consequence of two things:
  1.  All of the validation is now done by the `gibbs` function so we didn't need to correct the table early. There is no performance consequence to this since any table that was correctable by this call is correctable later in `validate_gibbs_input`.
  2.  Rounding seems better than ceiling - I think you and @lkursell made this argument at some point (or we discussed it). Differences are not going to happen because of rounding up or down by 1, and Dan originally rounded rather than ceiling.

I think rounding makes more sense than the ceiling calls we were doing. You had made this argument a while back (as had @lkursell), and after some thou